controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -439,6 +455,7 @@ columns:
     width: 10
   - id: Udns
     name: UDN labels
+    tooltip: The list of User Defined Networks.
```

Comment:
while u are here can u increase the width of this field to `15`

Reply:
done